src/app/components/login/login.component.html

```diff
@@ -18,6 +18,7 @@ <h5 class="modal-title">Access the platform</h5>
                     <input
                         [(ngModel)]="usuarioLogin.email"
                         type="text" value="" id="email-login" class="form-control"
+                        (keydown.enter) = "fazerLogin()"
```

Comment:
We only want the `RETURN` key to launch login in the password field, not the email.